docs/SSH_HOSTKEYS.md

```diff
@@ -0,0 +1,45 @@
+# SSH host keys configuration
+
+This example will go through using a TPM 2.0 for the openssh sshd host keys.
+As the ssh daemon supports using the SSH agent protocol we will use tpm2-pkcs11 with an ssh-agent.
+
+# Create a store and key
+The PIN is currently needed as ssh-agent doesn't seem to support an empty string PIN (just no PIN at all)
+```
+# mkdir /tmp/teststore-pkcs11
+# tpm2_ptool init --path=/tmp/teststore-pkcs11
+# tpm2_ptool addtoken --pid=1 --label=hostkey --sopin="mysopin" --userpin="myuserpin" --path=/tmp/teststore-pkcs11
+# tpm2_ptool addkey --algorithm=rsa2048 --label=hostkey --userpin="myuserpin" --path=/tmp/teststore-pkcs11
+```
+
+# Run a ssh agent
+The ssh agent should have a specific path for its socket, lets use /tmp/hostagent.sock for this example.
+```
+# export TPM2_PKCS11_STORE=/tmp/teststore-pkcs11 
+# ssh-agent -a /tmp/hostagent.sock
+```
+
+# Add the token to the agent
+```
+# SSH_AUTH_SOCK=/tmp/hostagent.sock ssh-add -s /usr/lib/x86_64-linux-gnu/libtpm2_pkcs11.so
+```
+
+# Get the public key in SSH format
+```
+# SSH_AUTH_SOCK=/tmp/hostagent.sock ssh-add -L
+ssh-rsa ....
```

Comment:
What's with the `....` is their arguments to the command that should go there?

Reply:
It's the output from ssh-add -L, I'll add a note about it

Reply:
@williamcroberts, I have updated it, please check it looks good